src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1130,65 +1130,117 @@ static LinkAnimationHeader* D_80853D7C[][2] = {
     { NA_SE_IT_SHIELD_POSTURE, ANIMSFX_DATA(ANIMSFX_TYPE_GENERAL, 16) },
     { NA_SE_IT_SHIELD_POSTURE, ANIMSFX_DATA(ANIMSFX_TYPE_GENERAL, 20) },
     { NA_SE_IT_SHIELD_POSTURE, -ANIMSFX_DATA(ANIMSFX_TYPE_GENERAL, 70) },
 };
 
-static AnimSfxEntry D_80853E34[] = {
+static AnimSfxEntry sIdleUnknownSfxEntry[] = {
     { NA_SE_IT_HAMMER_SWING, ANIMSFX_DATA(ANIMSFX_TYPE_GENERAL, 10) },
     { NA_SE_VO_LI_AUTO_JUMP, ANIMSFX_DATA(ANIMSFX_TYPE_VOICE, 10) },
     { NA_SE_IT_SWORD_SWING, ANIMSFX_DATA(ANIMSFX_TYPE_GENERAL, 22) },
     { NA_SE_VO_LI_SWORD_N, -ANIMSFX_DATA(ANIMSFX_TYPE_VOICE, 22) },
 };
 
-static AnimSfxEntry D_80853E44[] = {
+static AnimSfxEntry sIdleSwordThrustSfxEntry[] = {
     { NA_SE_IT_SWORD_SWING, ANIMSFX_DATA(ANIMSFX_TYPE_GENERAL, 39) },
     { NA_SE_VO_LI_SWORD_N, -ANIMSFX_DATA(ANIMSFX_TYPE_VOICE, 39) },
 };
 
-static AnimSfxEntry D_80853E4C[] = {
+static AnimSfxEntry sIdleRelaxSfxEntry[] = {
     { NA_SE_VO_LI_RELAX, -ANIMSFX_DATA(ANIMSFX_TYPE_VOICE, 20) },
 };
 
-static AnimSfxEntry* D_80853E50[] = {
-    D_80853DEC, D_80853DF0, D_80853DF4, D_80853DF8, D_80853DFC, D_80853E10,
-    D_80853E28, D_80853E34, D_80853E44, D_80853E4C, NULL,
+typedef enum {
+    /* 0x0 */ IDLE_ANIMATION_SFX_ENTRY_NONE,
+    /* 0x1 */ IDLE_ANIMATION_SFX_ENTRY_SNEEZE,
+    /* 0x2 */ IDLE_ANIMATION_SFX_ENTRY_SWEAT,
+    /* 0x3 */ IDLE_ANIMATION_SFX_ENTRY_HEAT_1,
+    /* 0x4 */ IDLE_ANIMATION_SFX_ENTRY_HEAT_2,
+    /* 0x5 */ IDLE_ANIMATION_SFX_ENTRY_BELT,
+    /* 0x6 */ IDLE_ANIMATION_SFX_ENTRY_FOOT_TAPPING,
+    /* 0x7 */ IDLE_ANIMATION_SFX_ENTRY_SHIELD_POSTURE,
+    /* 0x8 */ IDLE_ANIMATION_SFX_ENTRY_UNKNOWN,
+    /* 0x9 */ IDLE_ANIMATION_SFX_ENTRY_SWORD_THRUST,
+    /* 0xA */ IDLE_ANIMATION_SFX_ENTRY_RELAX,
+    /* 0xB */ IDLE_ANIMATION_SFX_ENTRY_MAX
+} IdleAnimationSfxEntryIndex;
+
+static AnimSfxEntry* sIdleSfxEntries[] = {
+    /* IDLE_ANIMATION_SFX_ENTRY_SNEEZE         */ sIdleSneezeSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_SWEAT          */ sIdleSweatSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_HEAT_1         */ sIdleHeat1SfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_HEAT_2         */ sIdleHeat2SfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_BELT           */ sIdleBeltSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_FOOT_TAPPING   */ sIdleFootTappingSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_SHIELD_POSTURE */ sIdleShieldPostureSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_UNKNOWN        */ sIdleUnknownSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_SWORD_THRUST   */ sIdleSwordThrustSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_RELAX          */ sIdleRelaxSfxEntry,
+    /* IDLE_ANIMATION_SFX_ENTRY_MAX            */ NULL,
 };
 
-static u8 D_80853E7C[] = {
-    0, 0, 1, 1, 2, 2, 2, 2, 10, 10, 10, 10, 10, 10, 3, 3, 4, 4, 8, 8, 5, 5, 6, 6, 7, 7, 9, 9, 0,
+static u8 sIdleSfxEntryIndices[] = {
+    /* gPlayerAnim_link_normal_wait_typeA_20f  */ IDLE_ANIMATION_SFX_ENTRY_NONE,
+    /* gPlayerAnim_link_normal_waitF_typeA_20f */ IDLE_ANIMATION_SFX_ENTRY_NONE,
```

Comment:
what do the comments correspond to?

Reply:
the animations that are assigned with the sfx entries, every idle animation has an entry on this array (plus an empty one at the end)